Hotfix to linebuffer to add flush

mbstrange2 opened the pull request:
Did this from my iPhone please fix spacing

kong0329 commented:
Max, when you flush, I think you should also flush data in `fifo_control`.
For example, `num_words_mem` in `fifo_control` should go 0 when you flush.
Here, you are only taking care of `linebuffer_control`.

mbstrange2 commented:
@kong0329 I added this flush in fifo_control.svp, I had refactored the num_words_mem to have more predictable action at reset and I forgot to flush it as well

kong0329 commented:
LGTM.
